AppCenter/AppCenterTests/Util/MSMockUserDefaults.m

```diff
@@ -26,6 +26,10 @@ - (instancetype)init {
   return self;
 }
 
+- (void)migrateKeys:(__unused NSDictionary *)migratedKeys forService:(__unused NSString *)serviceName {
+  // stub
```

Comment:
Remove it or revise the comment. Don't forget to add a line break before the comment.